### DISCUSSION TITLE
How do I render urls to open in new tab?

### DISCUSSION THREAD

**Discussion by fayazara** [nuxt/content#2668] [Q&A] [answered]
Is there a way to render some urls in a new tab instead of opening in the current tab?

**Comment by nobkd** [comment, marked as answer]
Maybe something like this? 

https://github.com/nuxt/content/issues/1783#issuecomment-1367301722

```md
[text](link){target="_blank"} 
```

**Reply by mzwoelfer** [reply]
That works fine. 
Is there a "global" solution for this?
I don't want to type it like that for every single link in my projects.

**Reply by hanyujie2002** [reply]
> That works fine. Is there a "global" solution for this? I don't want to type it like that for every single link in my projects.

Just override it with `ProseA.vue`. See https://content.nuxt.com/components/prose#prosea

1. Create `components/content/ProseA.vue`
2. Type code below (Based on [source code of ProseA](https://github.com/nuxt-modules/mdc/blob/main/src/runtime/components/prose/ProseA.vue)):

```vue
<template>
  <NuxtLink
    :href="href"
    :target="target"
  >
    <slot />
  </NuxtLink>
</template>

<script setup lang="ts">
import type { PropType } from 'vue'

defineProps({
  href: {
    type: String,
    default: ''
  },
  target: {
    type: String as PropType<'_blank' | '_parent' | '_self' | '_top' | (string & object) | null | undefined>,
    default: '_blank',
    required: false
  }
})
</script>
```

I changed the default value of `target`
